env.jenkins

```diff
@@ -1,23 +1,38 @@
 export AGENCY_API_HTTP_PORT=$(randport)
+
 export ATTACHMENT_SERVICE_REPL_PORT=$(randport)
 export ATTACHMENT_SERVICE_RPC_PORT=$(randport)
+
 export AUDIT_API_HTTP_PORT=$(randport)
+
+export AUDIT_SERVICE_REPL_PORT=$(randport)
```

Comment:
Why is this file even needed still? Because not everything is using `jest` yet?

Reply:
Yep!